web/templates/web/studies-list.html

```diff
@@ -56,27 +56,32 @@
 {% for study in object_list %}
 <div class="panel panel-default study-list">
     <div class="panel-body">
+        <div class="row">
+            <div class="col-sm-10 col-xs-12">
+                <h1>{{study.name}}</h1>
+                <p><em>{{study.lab.name}} ({{study.lab.institution}})</em></p>
+            </div>
+            <div class="col-sm-2 col-xs-12">
+                <div class="text-center">
+                    <a class="btn {% if study.eligible_children %}btn-success{% endif %} btn-default text-center" href="{% url 'web:study-detail' uuid=study.uuid %}" role="button">
+                        {% trans "See details" %}
+                    </a>
+                </div>
+            </div>
```

Comment:
This might be a bit too knick picky but, the "see details" button is a bit tight on the bottom when the window is very small.  If this is intended, then I can get over it. 

<img width="829" alt="Screen Shot 2021-07-01 at 1 53 33 PM" src="https://user-images.githubusercontent.com/44074998/124169063-c3712b00-da73-11eb-9c7f-50aa2e644254.png">

Reply:
Thanks for catching that! Here's what it looks like now:

![Screen Shot 2021-07-01 at 3 25 46 PM](https://user-images.githubusercontent.com/9554627/124179328-b73f9a80-da80-11eb-8c19-0b5ee1dc4833.png)

Reply:
I like it.